pulpcore/cli/ostree/__init__.py

```diff
@@ -1,12 +1,16 @@
 from typing import Any
 
 import click
+from pulp_glue.common.i18n import get_translation
 from pulpcore.cli.common.generic import pulp_group
 
 from pulpcore.cli.ostree.distribution import distribution
 from pulpcore.cli.ostree.remote import remote
 from pulpcore.cli.ostree.repository import repository
 
+translation = get_translation(__package__)
+_ = translation.gettext
```

Comment:
technically not _needed_, but I'm looking forward to the slovakian translation files. ;)